paymentsheet/res/layout/stripe_google_pay_button.xml

```diff
@@ -1,44 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 <merge xmlns:android="http://schemas.android.com/apk/res/android">
-
     <RelativeLayout
         android:id="@+id/google_pay_button_layout"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/stripe_paymentsheet_googlepay_button_height"
+        android:layout_height="wrap_content"
         android:paddingTop="2sp"
-        android:background="@drawable/googlepay_button_background"
         android:contentDescription="@string/google_pay">
-
         <com.stripe.android.paymentsheet.ui.PrimaryButton
             android:id="@+id/google_pay_primary_button"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
+            android:layout_height="@dimen/stripe_paymentsheet_googlepay_button_height"
+            android:layout_marginTop="@dimen/stripe_paymentsheet_googlepay_button_margin"
+            android:layout_marginHorizontal="@dimen/stripe_paymentsheet_googlepay_button_margin"
```

Comment:
Does the `com.google.android.gms.wallet.button.PayButton` feature internal padding or why are we adding this here?

Reply:
Yes, it also has internal padding that we need to account for 😢